.github/workflows/ci.yml

```diff
@@ -532,15 +532,19 @@ jobs:
 
   cross-test-with-parking_lot:
```

Comment:
Test for aarch64 is no longer "cross-test", but I leave it as is for now, the new runner is still in public preview, and if something goes wrong we can revert this PR with less-conflicts.